### ISSUE TITLE
Status unknown

### ISSUE THREAD

**Issue by TheFes** [pippyn/Home-Assistant-Sensor-Afvalbeheer#143]
The status of my trash containers is shown as Unknown as of some time today.
I guess this is related to the issue the Afvalwijzer integration has
https://github.com/xirixiz/Home-Assistant-Sensor-Afvalwijzer/issues/68

Something seems to be changed on afvalwijzer.nl in the way they handle json (as far as I understood in that issue)

**Comment by pippyn** [comment]
Do you also have the issue of HASS crashing? or only sensors show unknown?

**Comment by TheFes** [comment]
No, no crashes here.
I meant the unknown status is probably caused by the change at mijnafvalwijzer.nl.

**Comment by TheFes** [comment]
BTW, these messages are shown in my log

```
2020-09-10 20:30:59 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/afvalbeheer/sensor.py", line 425, in async_update
    await self.collector.update()
  File "/config/custom_components/afvalbeheer/sensor.py", line 552, in update
    response = r.json()
  File "/usr/local/lib/python3.8/site-packages/requests/models.py", line 898, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/simplejson/__init__.py", line 525, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.8/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/usr/local/lib/python3.8/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by pippyn** [comment]
I saw that too... not sure if the API the sensor uses is down for good or just temporarily ... Will do some research and keep you posted

**Comment by TheFes** [comment]
I also tried the afvalinfo integration (expecting the result would be the same) but somehow that integration manages to get at least the data of one of the two duobakken we use here.

https://github.com/heyajohnny/afvalinfo/issues/120

**Comment by pippyn** [comment]
That sensor doesn't use the API, but is just a website scraper

**Comment by TheFes** [comment]
Ah okay! Then it doesn't scrape very well ;-)

**Comment by sobo82** [comment]
Same problem here

**Comment by HammerNL89** [comment]
> I saw that too... not sure if the API the sensor uses is down for good or just temporarily ... Will do some research and keep you posted

Don't know if this helps, from the 'other' integration: 

https://github.com/xirixiz/homeassistant-afvalwijzer/issues/72#issuecomment-691542539

**Comment by pippyn** [comment]
Fixed in v4.6.4
